infra.bs

```diff
@@ -1694,6 +1694,16 @@ unordered one, since interoperability requires that any developer-exposed enumer
 contents be consistent between browsers. In those cases where order is not important, we still use
 ordered sets; implementations can optimize based on the fact that the order is not observable.
 
+<div algorithm>
+<p>To <dfn export for=set>create a set</dfn>, given a [=/list=] |input|:
```

Comment:
Hmm, I'd prefer naming this either `<dfn export>create a set from a list</dfn>`, or `<dfn export for="set">create</dfn>`. The latter makes sense if we think there's no other reasonable input for creating a set, whereas the former is more specific.

Reply:
Went for `<dfn export for="set">create</dfn>`. I think creating from other data structures is less common. Nevertheless, the create steps could accept a generic iterator instead of a list but since that is not currently specified it probably makes sense to just use the list.